<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="0" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-13T12:38:35Z" name="Default suite" finished-at="2024-05-13T12:38:36Z" duration-ms="1066">
    <groups>
    </groups>
    <test started-at="2024-05-13T12:38:35Z" name="Default test" finished-at="2024-05-13T12:38:36Z" duration-ms="1066">
      <class name="org.rahulshettyacademy.eCommersetc_4">
        <test-method is-config="true" signature="configureAppium()[pri:0, instance:org.rahulshettyacademy.eCommersetc_4@32c4e8b2]" started-at="2024-05-13T12:38:35Z" name="configureAppium" finished-at="2024-05-13T12:38:36Z" duration-ms="251" status="FAIL">
          <exception class="java.net.MalformedURLException">
            <message>
              <![CDATA[Error at index 1 in: "1:4723"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException: Error at index 1 in: "1:4723"
at java.base/java.net.URL.<init>(URL.java:708)
at java.base/java.net.URL.<init>(URL.java:569)
at java.base/java.net.URL.<init>(URL.java:516)
at io.appium.java_client.service.local.AppiumDriverLocalService.<init>(AppiumDriverLocalService.java:84)
at io.appium.java_client.service.local.AppiumServiceBuilder.createDriverService(AppiumServiceBuilder.java:473)
at io.appium.java_client.service.local.AppiumServiceBuilder.createDriverService(AppiumServiceBuilder.java:57)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:498)
at com.rahulsheetyacademy.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:56)
at org.rahulshettyacademy.TestUtils.AndroidBaseTest.configureAppium(AndroidBaseTest.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.NumberFormatException: Error at index 1 in: "1:4723"
at java.base/java.lang.NumberFormatException.forCharSequence(NumberFormatException.java:83)
at java.base/java.lang.Integer.parseInt(Integer.java:751)
at java.base/java.net.URLStreamHandler.parseURL(URLStreamHandler.java:237)
at java.base/java.net.URL.<init>(URL.java:703)
... 34 more
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configureAppium -->
        <test-method is-config="true" signature="preSetUp()[pri:0, instance:org.rahulshettyacademy.eCommersetc_4@32c4e8b2]" started-at="2024-05-13T12:38:36Z" name="preSetUp" finished-at="2024-05-13T12:38:36Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSetUp -->
        <test-method signature="fillForm(java.util.HashMap)[pri:0, instance:org.rahulshettyacademy.eCommersetc_4@32c4e8b2]" started-at="2024-05-13T12:38:36Z" name="fillForm" data-provider="getData" finished-at="2024-05-13T12:38:36Z" duration-ms="0" status="SKIP">
          <exception class="java.net.MalformedURLException">
            <message>
              <![CDATA[Error at index 1 in: "1:4723"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException: Error at index 1 in: "1:4723"
at java.base/java.net.URL.<init>(URL.java:708)
at java.base/java.net.URL.<init>(URL.java:569)
at java.base/java.net.URL.<init>(URL.java:516)
at io.appium.java_client.service.local.AppiumDriverLocalService.<init>(AppiumDriverLocalService.java:84)
at io.appium.java_client.service.local.AppiumServiceBuilder.createDriverService(AppiumServiceBuilder.java:473)
at io.appium.java_client.service.local.AppiumServiceBuilder.createDriverService(AppiumServiceBuilder.java:57)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:498)
at com.rahulsheetyacademy.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:56)
at org.rahulshettyacademy.TestUtils.AndroidBaseTest.configureAppium(AndroidBaseTest.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.NumberFormatException: Error at index 1 in: "1:4723"
at java.base/java.lang.NumberFormatException.forCharSequence(NumberFormatException.java:83)
at java.base/java.lang.Integer.parseInt(Integer.java:751)
at java.base/java.net.URLStreamHandler.parseURL(URLStreamHandler.java:237)
at java.base/java.net.URL.<init>(URL.java:703)
... 34 more
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillForm -->
        <test-method is-config="true" signature="preSetUp()[pri:0, instance:org.rahulshettyacademy.eCommersetc_4@32c4e8b2]" started-at="2024-05-13T12:38:36Z" name="preSetUp" finished-at="2024-05-13T12:38:36Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSetUp -->
        <test-method signature="fillForm(java.util.HashMap)[pri:0, instance:org.rahulshettyacademy.eCommersetc_4@32c4e8b2]" started-at="2024-05-13T12:38:36Z" name="fillForm" data-provider="getData" finished-at="2024-05-13T12:38:36Z" duration-ms="0" status="SKIP">
          <exception class="java.net.MalformedURLException">
            <message>
              <![CDATA[Error at index 1 in: "1:4723"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException: Error at index 1 in: "1:4723"
at java.base/java.net.URL.<init>(URL.java:708)
at java.base/java.net.URL.<init>(URL.java:569)
at java.base/java.net.URL.<init>(URL.java:516)
at io.appium.java_client.service.local.AppiumDriverLocalService.<init>(AppiumDriverLocalService.java:84)
at io.appium.java_client.service.local.AppiumServiceBuilder.createDriverService(AppiumServiceBuilder.java:473)
at io.appium.java_client.service.local.AppiumServiceBuilder.createDriverService(AppiumServiceBuilder.java:57)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:498)
at com.rahulsheetyacademy.utils.AppiumUtils.startAppiumServer(AppiumUtils.java:56)
at org.rahulshettyacademy.TestUtils.AndroidBaseTest.configureAppium(AndroidBaseTest.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.NumberFormatException: Error at index 1 in: "1:4723"
at java.base/java.lang.NumberFormatException.forCharSequence(NumberFormatException.java:83)
at java.base/java.lang.Integer.parseInt(Integer.java:751)
at java.base/java.net.URLStreamHandler.parseURL(URLStreamHandler.java:237)
at java.base/java.net.URL.<init>(URL.java:703)
... 34 more
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillForm -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:org.rahulshettyacademy.eCommersetc_4@32c4e8b2]" started-at="2024-05-13T12:38:36Z" name="tearDown" finished-at="2024-05-13T12:38:36Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.rahulshettyacademy.eCommersetc_4 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
